signbank/static/js/gloss_edit.js

```diff
@@ -255,11 +255,16 @@ function configure_edit() {
 
      $('.edit_list').on('click', function() {
          var choices = choice_lists[$(this).attr('id')];
+         var current_list = $(this)[0].id;
+         if (current_list == 'signer') {
```

Comment:
Can you do this via a class instead so we can reuse this behaviour?

```suggestion
         if ($(this).hasClass("edit_list_optional")) {
```
And then add the `edit_list_optional` class to your markup. That way, any other fields that need a 'none' option can use this class.